snmp_dell/README.md

```diff
@@ -0,0 +1,39 @@
+# Agent Check: snmp_Dell
+
+## Overview
+
+This check monitors [Dell][1].
+
+Dell Poweredge
+Dell Idrac
+Dell Isilon
+
+For details of monitored metrics see the [Network Device Monitoring][2] documentation.
+
+## Setup
+
+To install and configure the SNMP integration, see the [Network Device Monitoring][2] documentation.
+
+## Data Collected
+
+### Metrics
+
+For details of monitored metrics see the [SNMP integration tile][2]
```

Comment:
I would swap this to link to the NDM documentation, rather than the SNMP tile -- IMO it's kind of weird to link one tile to another, when we have the list of metrics in the docs as well